opencsp/common/lib/cv/spot_analysis/SpotAnalysisOperablesStream.py

```diff
@@ -25,37 +25,31 @@ def __init__(
         self, images: ImagesIterable | ImagesStream | SpotAnalysisImagesStream | Iterator[SpotAnalysisOperable]
     ):
         """
+        Initializes the SpotAnalysisOperablesStream with the provided image source.
+
         Parameters
         ----------
         images : ImagesIterable | ImagesStream | SpotAnalysisImagesStream | Iterator[SpotAnalysisOperable]
-            The images stream to be used as the primary images for the produced
-            operables. This stream will be restartable as long as the given
-            'images' stream is restartable.
+            The source of images to be processed.
```

Comment:
Please keep the original comments.

```suggestion
            The source of images to be processed. This will be used as the
            primary images for the produced operables. This
            SpotAnalysisOperablesStream stream will be restartable so long as
            the given 'images' stream is also restartable.
```